styles/mixins/_focus.scss

```diff
@@ -2,6 +2,11 @@
 
 $-color-focus-outline: var.$color-focus-outline;
 
+@mixin outline-rule($inset: false) {
+  outline: none;
+  box-shadow: 0 0 0 2px $-color-focus-outline if($inset, inset, null);
```

Comment:
I am so behind with scss 😢 . I really need to learn more, for example here `if(....)`

Reply:
Hah! This isn't mine, actually; it's just a moved rule that pre-exists these changes. I also have a little trouble human-parsing the `if...` here!